src/components/navbar.js

```diff
@@ -30,10 +32,10 @@ function NavBar(){
             </InputGroup.Prepend>
             </InputGroup>
           </Form>
-          <Nav>
-            <Nav.Link  href="#signin" active ><Button variant="success"><Icon.BoxArrowInRight/> Sign In</Button></Nav.Link>
-            <Nav.Link  href="#signup" active ><Button variant="warning"><Icon.PersonPlus/> Sign Up</Button></Nav.Link>
-          </Nav>
+          {/* <Nav> */}
+            <Link  to="/Signin" ><Button style={{marginLeft: '10px', marginRight:'10px'}} variant="success"><Icon.BoxArrowInRight/> Sign In</Button></Link>
```

Comment:
Don't use hard code the values in pixels because that makes the website unresponsive, instead use relative units of measurement such as vh or vw or rem.